lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -463,9 +463,11 @@ local table_Copy = table.Copy
 
 -- Helper function for gmatch (below)
 -- (By Divran)
-local DEFAULT = {n={},ntypes={},s={},stypes={},size=0,istable=true,depth=0}
+local newE2Table = E2Lib.newE2Table
+-- It seems we don't need 'istable' and 'depth' params in an e2 table. So we are going to disregard them.
```

Comment:
this is correct, these variables have since been removed. you can also remove this comment explaining that they have been removed

Reply:
oh yeah, you usually explain changes in the commit logs.
But we squash those, making them almost unusable.
Maybe we should rethink that policy, @Divran :)